Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/DeleteOrphanManifestNodesTask.java

```diff
@@ -0,0 +1,116 @@
+    })
+    public void run() {
+        progress.start(Bundle.ManifestNodesCleanupTask_progress_startMessage());
+        try {
+            progress.progress(Bundle.ManifestNodesCleanupTask_progress_connectingToCoordSvc());
+            logger.log(Level.INFO, "Connecting to the coordination service for orphan manifest file node clean up");  // NON-NLS            
+            CoordinationService coordinationService;
+            try {
+                coordinationService = CoordinationService.getInstance();
+            } catch (CoordinationService.CoordinationServiceException ex) {
+                logger.log(Level.WARNING, "Error connecting to the coordination service", ex); // NON-NLS
+                return;
+            }
+
+            progress.progress(Bundle.ManifestNodesCleanupTask_progress_gettingCaseNodesListing());
+            logger.log(Level.INFO, "Querying coordination service for case nodes for orphaned case node clean up");  // NON-NLS
+            List<AutoIngestJobNodeData> nodeDataList;
+            try {
+                nodeDataList = AutoIngestJobNodeDataCollector.getNodeData();
+            } catch (CoordinationService.CoordinationServiceException ex) {
+                logger.log(Level.WARNING, "Error collecting auto ingest job node data", ex); // NON-NLS
+                return;
+            } catch (InterruptedException ex) {
+                logger.log(Level.WARNING, "Unexpected interrupt while collecting auto ingest job node data", ex); // NON-NLS
+                return;
+            }
+
+            for (AutoIngestJobNodeData nodeData : nodeDataList) {
+                final String caseName = nodeData.getCaseName();
+                final Path manifestFilePath = nodeData.getManifestFilePath();
+                final File manifestFile = manifestFilePath.toFile();
+                if (!manifestFile.exists()) {
+                    try {
+                        progress.progress(Bundle.ManifestNodesCleanupTask_progress_deletingOrphanedManifestNode(manifestFilePath));
+                        logger.log(Level.INFO, String.format("Deleting orphaned case node %s for %s", manifestFilePath, caseName));  // NON-NLS
+                        coordinationService.deleteNode(CoordinationService.CategoryNode.MANIFESTS, manifestFilePath.toString());
+                    } catch (CoordinationService.CoordinationServiceException ex) {
+                        if (!DeleteCaseUtils.isNoNodeException(ex)) {
+                            logger.log(Level.SEVERE, String.format("Error deleting %s znode for %s", manifestFilePath, caseName), ex);  // NON-NLS
+                        }
+                    } catch (InterruptedException ex) {
+                        logger.log(Level.WARNING, String.format("Unexpected interrupt while deleting %s znode for %s", manifestFilePath, caseName), ex);  // NON-NLS
```

Comment:
Does this need a stack trace just because the user canceled the process? It's not really 'unexpected' either as we give them a cancellation button.

Reply:
It is indeed not unexpected, nor is the stack trace helpful.